admin-dev/themes/new-theme/scss/components/_multishop-selector.scss

```diff
@@ -0,0 +1,180 @@
+// stylelint-disable-next-line
+$component-name: shop-selector;
+
+.#{$component-name} {
+  &-search {
+    min-height: auto;
+    padding: 0.3rem 1rem;
+    padding-left: 2.2rem;
+
+    &-container {
+      position: relative;
+      margin-bottom: 1rem;
+
+      @include media-breakpoint-down(sm) {
+        display: none;
+      }
+
+      i {
+        position: absolute;
+        top: 0.5rem;
+        left: 0.75rem;
+        z-index: 1;
+        font-size: 1.2rem;
+        color: $medium-gray;
+        pointer-events: none;
+      }
+
+      .tt-menu {
+        width: 100%;
+        overflow: hidden;
+        background-color: $white;
+        box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.1);
+        @include border-bottom-radius(4px);
+
+        .tt-dataset > div {
+          padding: 0.5rem 1rem;
+          cursor: pointer;
+          @include transition(0.25s ease-out);
+
+          &:hover,
+          &.tt-cursor {
+            background-color: #f4fcfd;
+          }
+        }
+      }
+    }
+  }
+
+  ul.#{$component-name}-group-list {
+    position: relative;
+    max-height: 500px;
+    padding: 0 1rem 0 0;
+    margin-left: 0;
+    list-style-type: none;
+  }
+
+  &-group-item {
+    &.#{$component-name}-item {
+      justify-content: flex-start;
+      margin-bottom: 1rem;
+      font-weight: 600;
+      color: $medium-gray;
+    }
+  }
+
+  &-item {
+    display: flex;
+    align-items: start;
+    justify-content: start;
+    width: 100%;
+
+    > .#{$component-name}-color-check .material-icons {
+      font-size: 1.25rem;
+    }
+
+    .md-checkbox {
+      width: 100%;
+      cursor: pointer;
+
+      label {
+        width: 100%;
+        padding-left: 25px;
+
+        .md-checkbox-control::before,
+        .md-checkbox-control::after {
+          margin-top: 10px;
+        }
+      }
+    }
+
+    &-name {
+      display: flex;
+      align-items: start;
+      width: 100%;
+      padding: 0.5rem;
+      font-weight: 600;
+      cursor: pointer;
+      @include border-radius(0.5rem);
+
+      .material-icons {
+        margin-left: auto;
+        font-size: 1.4rem;
+        color: $brand-primary;
+        opacity: 0;
+      }
+
+      .shop-selector-status {
+        margin-right: 5px;
+        margin-left: auto;
+        color: $brand-primary;
+      }
+    }
+
+    &-name:hover {
+      background-color: rgba(37, 185, 215, 0.15);
```

Comment:
`37, 185, 215` could be replaced by an SCSS var, even if it's a close one and not the exact same color